uievents/click/dblclick_event_mouse.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>dblclick event for the mouse pointer type</title>
+    <link rel="author" title="Google" href="http://www.google.com/" />
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <style>
+    #target
+    {
+        background-color: green;
+        width: 200px;
+        height: 200px;
+    }
+    </style>
+  </head>
+  <body>
+    <p>Double-click on the green box with the left mouse button.</p>
+    <div id="target"></div>
+    <script>
+        setup({ single_test: true });
+        var elem = document.getElementById("target");
+        elem.addEventListener("dblclick", function(e) {
+            assert_equals(e.target.id, "target");
+            assert_equals(e.detail, 2);
+            done();
```

Comment:
I believe this may cause some leak in Chrome bots. @SummerLW for more context.
We might need to still wait for the test_driver promise to get resolved as well before calling done here.

Reply:
Aha, so waiting for that matters even in a single test? I thought it was to make tests work when there were multiple subtests in a file, but yeah I can restore the waiting.

Reply:
I've restored this now using `t.add_cleanup`